rules/auth0_rules/auth0_user_invitation_created.py

```diff
@@ -0,0 +1,43 @@
+import re
+
+from global_filter_auth0 import filter_include_event
+from panther_auth0_helpers import auth0_alert_context, is_auth0_config_event
+from panther_base_helpers import deep_get
+
+org_re = re.compile(r"^/api/v2/organizations/[^/\s]+/invitations$")
+
+
+def rule(event):
+    if not filter_include_event(event) or not is_auth0_config_event(event):
+        return False
+
+    return invitation_type(event) is not None
+
+
+def title(event):
+    inv_type = invitation_type(event)
+    if inv_type == "tenant":
+        invitee = deep_get(event, "data", "details", "request", "body", "owners")[0]
+    elif inv_type == "organization":
+        invitee = deep_get(event, "data", "details", "request", "body", "invitee", "email")
+    else:
+        raise Exception(f"Unexpected invitation type {inv_type}")
+
+    inviter = deep_get(event, "data", "details", "request", "auth", "user", "email")
+    source = deep_get(event, "p_source_label", default="unknown")
+    return f"Auth0 User [{inviter}] invited [{invitee}] to {inv_type.title()} [{source}]]"
+
+
+def invitation_type(event):
+    path = deep_get(event, "data", "details", "request", "path")
```

Comment:
```suggestion
    path = deep_get(event, "data", "details", "request", "path", default="")
```

Line 36 will raise a TypeError if path is `None`, which is the default for `deep_get` when no default=kwarg is provided or the item either doesn't exist or it does exist and has a `null` value in the inbound log

```python
user@computer:~/Sources/PAN/panther-analysis $ python3
Python 3.11.3 (main, Apr  7 2023, 20:13:31) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import re
>>> org_re = re.compile(r"^/api/v2/organizations/[^/\s]+/invitations$")
>>> org_re.match(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: expected string or bytes-like object, got 'NoneType'
```

Reply:
good call